src/adapter-types/src/dyncfgs.rs

```diff
@@ -55,8 +55,8 @@ pub const WITH_0DT_CAUGHT_UP_CHECK_ALLOWED_LAG: Config<Duration> = Config::new(
 
 pub const WITH_0DT_CAUGHT_UP_CHECK_CUTOFF: Config<Duration> = Config::new(
     "with_0dt_caught_up_check_cutoff",
-    Duration::from_secs(10 * 60), // 10 minutes
-    "During a 0dt deployment, if a cluster has only 'problematic' (crash-looping) replicas _and_ any collection that is behind by more than this cutoff, the cluster will be ignored in caught-up checks.",
+    Duration::from_secs(2 * 60 * 60), // 2 hours
```

Comment:
Changed this back to the old default, since cutting over too soon is worse than cutting over too late. A value between the two might work as well, but not sure how to choose one.